mtp_cashbook/templates/disbursements/start.html

```diff
@@ -11,23 +11,34 @@
   <div class="mtp-disbursement-form-preview grid-row">
     <div class="column-two-thirds">
       <header>
-        <h1 class="heading-xlarge">{% trans 'Digital disbursements' %}</h1>
+        <h1 class="heading-xlarge">{% trans 'Digital disbursements tool' %}</h1>
       </header>
+      <h3><strong>{% trans 'How does it work' %}</strong></h3>
+      {% blocktrans trimmed %}
```

Comment:
we need to wrap each sentence separately. each translation string must be as short as possible while being and entire unit of sense. like a title, a sentence